src/test/mochitest/browser_dbg-breakpoints.js

```diff
@@ -73,3 +73,37 @@ add_task(async function() {
   bp2 = findBreakpoint(dbg, "simple2", 5);
   is(bp2.disabled, false, "second breakpoint is enabled");
 });
+
+add_task(async function() {
+  Services.prefs.setBoolPref("devtools.features.skip-pausing", true);
```

Comment:
i believe we want to use pushPref so it resets at the end

https://github.com/jasonlaster/debugger.html/blob/1b737fc00f7356e9ea5ceda3a5938f9924d4d39f/src/test/mochitest/browser_dbg-async-stepping.js#L6-L7

Reply:
Ohhh, nice!